src/app/static/src/js/components/runWorkflow/runWorkflow.vue

```diff
@@ -18,7 +18,7 @@
             <a href="#" @click.prevent="$emit('view-progress', createdWorkflowKey)">View workflow progress</a>
         </div>
         <div class="pt-4 col-sm-6">
-            <error-info :default-message="defaultMessage" :api-error="error"></error-info>
+            <error-info default-message="An error occurred while running the workflow" :api-error="error"></error-info>
```

Comment:
Vue throwing an error here because this property is not defined on the Vue instance. Since it never actually changes it might as well just be a static string.